packages/@aws-cdk/aws-scheduler-alpha/lib/schedule.ts

```diff
@@ -59,6 +59,47 @@ export interface ScheduleTargetProps {
   readonly retryAttempts?: number;
 }
 
+/**
+ * A time window during which EventBridge Scheduler invokes the schedule.
+ */
+export class FlexibleTimeWindowMode {
+  /**
+   * FlexibleTimeWindow is disabled.
+   */
+  public static off(): FlexibleTimeWindowMode {
```

Comment:
I changed it to camelCase because it resulted in an error during cdk-build.
```
error JSII8002: Method and property (unless they are static readonly) names must use camelCase. Rename "@aws-cdk/aws-scheduler-alpha.FlexibleTimeWindowMode.FLEXIBLE" to "flexible"
error JSII8002: Method and property (unless they are static readonly) names must use camelCase. Rename "@aws-cdk/aws-scheduler-alpha.FlexibleTimeWindowMode.OFF" to "off"
```